infra/docker/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN mkdir -p /build \
   && cd /build \
   && git clone https://github.com/NemProject/nem.git
 
-# actually build (using java 8)
+# actually build (using java 11)
 RUN echo 2 | update-alternatives --config java \
```

Comment:
why do you need `echo 2 | update-alternatives --config java` if you're using latest java?